common/build/eslint-plugin-fluid/PUBLISH_GUIDE.md

```diff
@@ -0,0 +1,86 @@
+   │   └── rule-two/				<!-- Test suite for rule-two -->
+   │       └── rule-two.test.js
+
+3. **Update Changelog**: Record the new rule in the `CHANGELOG.md` file of the `@fluid-internal/eslint-plugin-fluid` package. This provides visibility into what was added for future reference.
+
+4. **Version Bump**: Update the package version following [Fluid Semantic Versioning](https://eng.ms/docs/experiences-devices/opg/office-shared/fluid-framework/fluid-framework-internal/fluid-framework/docs/dev/wiki/sync/breaking-vs-non-breaking-changes) guidelines. Typically:
+   - **Patch** version for fixes
+   - **Minor** version for new rules (backward-compatible)
+   - **Major** version for breaking changes
+
+### 2. Publish New Version of `eslint-plugin-fluid`
+
+Once PR 1 is merged, publish the new version of `@fluid-internal/eslint-plugin-fluid` by following the steps:
+
+1. **Publish**: Follow the [guide](https://eng.ms/docs/experiences-devices/opg/office-shared/fluid-framework/fluid-framework-internal/fluid-framework/docs/on-call/release/release) to publish the new version of `@fluid-internal/eslint-plugin-fluid`.
+
+2. **Verify Release**: Confirm that the release was successful by checking the package version on the [NPM Registry](https://www.npmjs.com/package/@fluid-internal/eslint-plugin-fluid).
+
+### 3. Update `eslint-config-fluid`'s Dependency on `eslint-plugin-fluid` (PR 2)
+
+In `@fluidframework/eslint-config-fluid`, update the version of `@fluid-internal/eslint-plugin-fluid` to the newly published version:
+
+1. **Bump Dependency**: Update the dependency in `package.json` to reference the latest version of `@fluid-internal/eslint-plugin-fluid`.
+
+2. **Open PR**: Submit a pull request for this update.
+
+### 4. Add New Rule to the Appropriate Config
+
+Depending on the scope of the rule, add it to one of the following configurations (NOTE: `recommended` extends thre `minimal-deprecated.js`):
+   - `minimal-deprecated.js`
+   - `recommended.js`
+   - `strict.js`
+
+1. **Update Changelog**: Record the change in `eslint-config-fluid`'s `CHANGELOG.md`.
+
+2. **Version Bump**: Update the version of `eslint-config-fluid`.
+
+3. **Fix Violations in the Repo**:
+   - Install the local version of `eslint-config-fluid` across relevant release groups.
+   - Run the linter to identify and fix any violations.
+   - To simplify integration, add the following to the `pnpmOverrides` section of `package.json`:
+     ```json
+     {
+       "pnpmOverrides": {
+         "@fluidframework/eslint-config-fluid": "file:<relative-path-to-config-directory>"
+       }
+     }
+     ```
+
+### 5. Publish New Version of `eslint-config-fluid`
+
+Once the PR is merged, publish the new version of `eslint-config-fluid` following the [guide](https://eng.ms/docs/experiences-devices/opg/office-shared/fluid-framework/fluid-framework-internal/fluid-framework/docs/on-call/release/release).
+
+
+### 6. Update Dependencies on `eslint-config-fluid` Across the Repo (PR 3)
```

Comment:
Still need details here looks like. Would probably be good to note that some new violations may have crept in since the previous PR that will need to be fixed.